src/components/style/media/avatar.css

```diff
@@ -0,0 +1,39 @@
+.fdsAvatar {
+  font-size: var(--font-size-xs);
+  user-select: none;
+  transition: background-color var(--button-timeout) var(--motion-easing-default),
```

Comment:
V tiny nitpick - we don't need to transition `background-color` on this

Reply:
Adjusting! Going to remove transition altogether now that we don't have the focus style.